### DISCUSSION TITLE
为什么不开源内存优化函数

### DISCUSSION THREAD

**Discussion by mc-124** [Meloong-Git/PCL#2500] [提问] [answered]
为什么没有开源PageOtherTest.MemoryOptimizeInternal()？

**Comment by SALTWOOD** [comment]
其实就是swap，简单（

**Reply by mc-124** [reply]
能告诉我大致怎么弄吗？我想给我新写的启动器加上这个功能，也是vb.net

**Reply by Chiloven945** [reply]
你要知道你正在最大的源代码开源共享平台Github上发送评论，且上面就有个Type `/` to search

**Comment by LTCatt** [comment, marked as answer]
Private Declare Function NtSetSystemInformation Lib "ntdll.dll" (SystemInformationClass As Integer, SystemInformation As IntPtr, SystemInformationLength As Integer) As UInteger
